source/release-notes.txt

```diff
@@ -2,6 +2,38 @@
 Release Notes
 =============
 
+MongoDB Connector for Spark 10.3
+--------------------------------
+
+The 10.3 connector release includes the following new features:
+
+- Added support for reading multiple collections when using micro-batch or
+  continuous streaming modes.
+
+  .. warning:: Breaking Change
+
+     Support for reading multiple collections introduces the following breaking
+     changes:
+     
+     - If the name of a collection used in your ``collection`` configuration
+       option contains a comma (,), the
```

Comment:
S: I _think_ we can get away without showing the comma in parentheses. The backslash at the end makes sense for clarity.
```suggestion
       option contains a comma, the
```